com.calibre_ebook.calibre.yaml

```diff
@@ -47,12 +58,6 @@ modules:
         sha256: 37715ef44a75d4ae7cf2fc9a379805837a29b44f1eeacfab853a0b4acd73157a
         size: 108232300
       - type: file
-        path: com.calibre_ebook.calibre.appdata.xml
-      - type: file
-        path: com.calibre_ebook.calibre.desktop
-      - type: file
-        path: com.calibre_ebook.calibre.ebook_edit.desktop
-      - type: file
-        path: com.calibre_ebook.calibre.ebook_viewer.desktop
-      - type: file
-        path: com.calibre_ebook.calibre.lrfviewer.desktop
+        path: fixup-exports.py
+    modules:
+      - xdg-utils.json
```

Comment:
The JSON file is not that big, let's inline the additional modules in YAML format here instead to keep information centralized in the manifest.

Reply:
Done!